app/src/androidTest/java/com/github/h3lp3rs/h3lp/LocalEmergencyCallTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.github.h3lp3rs.h3lp
```

Comment:
Since these tests don't have to run on the emulator, shouldn't they be in the test folder instead of the androidTest folder?

Reply:
That is indeed what I wanted to do at first, the problem comes from the fact that both the Geocoder and the reading of the emergency numbers database requires the application context which I can't seem to be able to access from a simple unit test (in the test folder). Consequently, I can't call the methods correctly to test them from the test folder. 
I found no solution online to get the context from the tests in the test folder (which I think makes sense, a unit test should not require access to the application context).